doc/tutorials/04-lattice_boltzmann/scripts/04-lattice_boltzmann_part3_solution.py

```diff
@@ -0,0 +1,109 @@
+import espressomd
+espressomd.assert_features(['LENNARD_JONES'])
```

Comment:
You might want to add tests for these new scripts. And remove the test for the deleted script.